vision/standard_object/odlc_contour_detection.py

```diff
@@ -82,15 +84,21 @@ def fetch_shape_contours(
     # White threshold is used to find the brightest parts of the image
     # Black threshold is used to find the darkest parts of the image
     # Saturation threshold is used to find the most saturated parts of the image
-    _, white_thresh = cv2.threshold(img_brightness, MIN_WHITE_VALUE, 255, cv2.THRESH_BINARY)
-    _, black_thresh = cv2.threshold(img_brightness, MAX_BLACK_VALUE, 255, cv2.THRESH_BINARY_INV)
-    _, saturation_thresh = cv2.threshold(img_saturation, saturation_value, 255, cv2.THRESH_BINARY)
+    white_thresh: Mask
+    black_thresh: Mask
+    saturation_thresh: Mask
+
+    _, white_thresh = cv2.threshold(blurred_brightness, MIN_WHITE_VALUE, 255, cv2.THRESH_BINARY)
+    _, black_thresh = cv2.threshold(blurred_brightness, MAX_BLACK_VALUE, 255, cv2.THRESH_BINARY_INV)
+    _, saturation_thresh = cv2.threshold(
+        blurred_saturation, saturation_value, 255, cv2.THRESH_BINARY
+    )
 
     # expands each threshold by 3 pixels in the x and y direction
     # to merge those in close proximity to one another
-    white_thresh = cv2.dilate(white_thresh, np.ones([3, 3], np.uint8))
-    black_thresh = cv2.dilate(black_thresh, np.ones([3, 3], np.uint8))
-    saturation_thresh = cv2.dilate(saturation_thresh, np.ones([3, 3], np.uint8))
+    white_thresh = DILATION_KERNEL
```

Comment:
`DILATION_KERNEL` only replaces the `np.ones(...)`, so:

```suggestion
    white_thresh = cv2.dilate(white_thresh, DILATION_KERNEL)
```